Dockerfile

```diff
@@ -56,19 +56,13 @@ RUN curl -sS -L https://github.com/facebook/buck/releases/download/v${BUCK_VERSI
 # Full reference at https://dl.google.com/android/repository/repository2-1.xml
 # download and unpack android
 RUN curl -sS https://dl.google.com/android/repository/${SDK_VERSION} -o /tmp/sdk.zip \
-    && mkdir /opt/android \
-    && unzip -q -d /opt/android /tmp/sdk.zip \
-    && rm /tmp/sdk.zip
-
-
-# Add android SDK tools
-RUN cd ~ && mkdir ~/.android && echo '### User Sources for Android SDK Manager' > ~/.android/repositories.cfg \
```

Comment:
We need the lines back to avoid install warning and other potential warning.

Reply:
there were warnings even with this line, so I removed it.